src/gapstate.h

```diff
@@ -195,19 +199,22 @@ extern GAPState MainGAPState;
 
 #endif
 
-void InitMainGAPState(void);
+typedef void * GAPModuleState;
+typedef void (*GAPModuleConstructor)();
+typedef void (*GAPModuleDestructor)();
+
+typedef struct {
+    Int Offset;
+    GAPModuleConstructor Constructor;
+    GAPModuleDestructor Destructor;
```

Comment:
We almost start `struct` member names in lower case (and that's the convention I usually see in other C projects). Of course `GAPState` is an exception, but that's mostly for historical reasons: we turned globals (which we start in upper case) into struct members, and for simplicity (? or out of laziness? :-) did not change their capitalization.

Hence I find it quite jarring to read `foo->Bar` instead of `foo->bar` in code.